### PR TITLE
Add Showcase section

### PR DESCRIPTION
**Pull request by colepeters**
Adds a Showcase section to the docs site, along with an initial selection of projects. Just waiting for confirmation from Tim Kadlec and Jonathan Lipps re: featuring their projects.

image

### CODE REVIEW

File: app/elements/doc/showcase.mjs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <article class="mt0 mb2 p0 p2-lg">
+      <a href="${url}" target="_blank">
+        <figure>
+          <img src="${image}" alt="" class="mb0" />
+        </figure>
+        <h1 class="font-medium">${title}</h1>
+        <p class="mt0 leading3">
+          <slot></slot>
+        </p>
+        <p class="mt0"><span class="font-medium">Author:</span> ${author}</p>
```
Comment (macdonst) [inline app/elements/doc/showcase.mjs:35]: Can we wrap the author's name in a url so it can take them to their socials or homepage?

**Review by macdonst — APPROVED** [review]
LGTM, just one question if we can get a link to socials or homepage for the authors of the apps.

File: app/elements/doc/showcase.mjs (hunk 1 above)
Comment (colepeters) [reply app/elements/doc/showcase.mjs:35]: Only issue with that is I'm wrapping the whole card in a link to make it all clickable. I could see about moving the author name out of the card to make it its own link tho!

Comment (macdonst) [reply app/elements/doc/showcase.mjs:35]: I am not married to the idea so no worries.

Comment (colepeters) [reply app/elements/doc/showcase.mjs:35]: It's a good idea, I'll take a look! :)

**Pull request merged**